witness/golang/internal/witness/witness.go

```diff
@@ -151,15 +132,15 @@ func (w *Witness) Update(ctx context.Context, logID string, nextRaw []byte, proo
 	// Get the latest one for the log because we don't want consistency proofs
 	// with respect to older checkpoints.  Bind this all in a transaction to
 	// avoid race conditions when updating the database.
-	tx, err := w.db.BeginTx(ctx, nil)
+	write, err := w.lsp.WriteOps(logID)
 	if err != nil {
-		return nil, fmt.Errorf("couldn't create db tx: %v", err)
+		return nil, fmt.Errorf("WriteOps(%v): %v", logID, err)
 	}
 	// Defer a rollback to clean up the TX if something fails.
```

Comment:
The comment's probably accurate about what's going on behind the abstraction, but might be worth updating to talk about the abstraction?

Reply:
Done.